cli/Cargo.toml

```diff
@@ -33,6 +33,7 @@ bytes = "0.5.3"
 byteorder = "1.3.2"
 brotli2 = "0.3.2"
 clap = "2.33.0"
+derive_deref = "1.1.0"
```

Comment:
This seems unrelated to the goal of the patch.

I’d prefer if we continued to impl Deref manually - it’s not much work and don’t like adding dependencies.

Reply:
I find myself writing trivial Deref/Derefmut impls so often that I recently just gave in.